.github/workflows/sprinkler.yml

```diff
@@ -33,6 +36,14 @@ jobs:
    steps:
      - name: Checkout Repository
        uses: actions/checkout@v3.0.2
+     - name: configure aws credentials
+       uses: aws-actions/configure-aws-credentials@v1
+       with:
+         role-to-assume: arn:aws:iam::${{ env.ACCOUNT_ID }}:role/github-actions
+         role-session-name: samplerolesession
+         aws-region: ${{ env.AWS_REGION }}
+       env:
+         ACCOUNT_ID: 348456244381 # sprinkler-development
```

Comment:
Can we find a way so that this is not hard coded please?

Reply:
I moved the account id to env variable. Not sure you would like this. I think it should be alright because, we pass other things as env variables as well like the region. The alternative would be to add it to a github secret however it could become messy to create a separate secret for every account id.